applications/RomApplication/tests/test_calculate_rom_basis_output_process.py

```diff
@@ -87,25 +88,14 @@ def __ExecuteTest(self, process_settings):
 
     def __CheckResults(self):
         with KratosUnittest.WorkFolderScope(self.work_folder, __file__):
-            # Set check process settings
-            settings = KratosMultiphysics.Parameters("""{
-                "reference_file_name"   : "",
-                "output_file_name"      : ""
-            }""")
             output_filename = "{}.{}".format(self.process_settings["rom_basis_output_name"].GetString(), self.process_settings["rom_basis_output_format"].GetString())
             reference_filename = "{}Results.{}".format(self.process_settings["rom_basis_output_name"].GetString(), self.process_settings["rom_basis_output_format"].GetString())
-            settings["output_file_name"].SetString(output_filename)
-            settings["reference_file_name"].SetString(reference_filename)
-
-            # Call the files comparison process
-            cmp_process = CompareTwoFilesCheckProcess(settings)
-            cmp_process.ExecuteInitialize()
-            cmp_process.ExecuteBeforeSolutionLoop()
-            cmp_process.ExecuteInitializeSolutionStep()
-            cmp_process.ExecuteFinalizeSolutionStep()
-            cmp_process.ExecuteBeforeOutputStep()
-            cmp_process.ExecuteAfterOutputStep()
-            cmp_process.ExecuteFinalize()
+            
+            with open(output_filename) as f:
+                output_data = json.load(f)
+            with open(reference_filename) as f:
+                reference_data = json.load(f)
+            self.assertEqual(output_data, reference_data)
```

Comment:
this again compares it without taking floating point numbers into account I think

Reply:
Sure... It is most probably related to this... Thanks. I'll think about.